### ISSUE TITLE
Problem "waiting for servers"

### ISSUE THREAD

**Issue by programgames** [TheLimePixel/GregBlock#103]
Hi everyone,
Me and my friends started playing on Gregblock one month ago,
Since this day everything was fine, until now , we are getting the error "Waiting for servers" every 5 minutes and we have to dc and reco to play again.
Since i've tought it was my hosting platform "Verygames" i added a tick profiler profiler mod, and we are getting 20 tps.
So i dediced to upgrade my plan to a very big plan  i7-7700 k (overclocked .. 3Go ram ).
None has been changed. This is the first time we play with gregtech ( and most mods included in this modpack). And we have already played on much more bigger modpack ( around 220 mods) and we had no problems until this modpack.
So , what could be the problem ?

TL:DR ( two pics of our base at this moment ) :  [https://i.imgur.com/WrY1wXs.png](url) [https://i.imgur.com/0MfrTK5.png](url)

**Comment by programgames** [comment]
Ok we found a part of the problem , we deleted these two boilers and the electic blast furnace , and everything was fine. We are goind to build them in one chunk separately and see what happens. I will tell if that will change ...

**Comment by programgames** [comment]
I found the problem, when an Input hatch ( LV ) is installed on the ground we are getting instant waiting for servers and after that we can't open any machines without waiting 60 s or disconnect. We will test with input hatch mv for the test.

**Comment by programgames** [comment]
Well that not changed the problem.. I will follow on gregtech community edition github

**Comment by programgames** [comment]
fixed in new version of gregblock , you can close the issue

**Comment by programgames** [comment]
Fixed in new version of gregblock
